### DISCUSSION TITLE
List of critical problems within the JSON Schema Ecosystem

### DISCUSSION THREAD

**Discussion by benjagm** [json-schema-org/community#578] [General]
**Do you feel that we have critical problems that needs to be solved for the benefit of the JSON Schema Ecosystem? 
Then you are not alone!** 

This is a call to action for the JSON Schema Community to collectively identify and prioritize the most significant challenges or problems that are still pending resolution within the JSON Schema Ecosystem. 

We invite everyone in the Community to share their insights and upvote options provided by others to identify together the most critical issues for the JSON Schema Ecosystem. Once we have clarity on the challenges, the next step is to create a benefits program to make it it easy for community members to create projects addressing these critical issues.

### 🎯 Identifying Critical Problems:
**How to Participate:**
1. Share: Comment with the critical problems you've encountered while working with JSON Schema.
2. Upvote: Upvote problems mentioned by others that resonate with your experiences.

**Why Your Input Matters:**
- By the Community For the Community: This process is driven by the Community to ensure we address the most pressing issues together.
- Prioritization: Upvotes help prioritize challenges, focusing on those affecting a larger portion of the Community.

**Some examples of projects solving critical problems:**
Projects like [Bowtie](https://github.com/bowtie-json-schema/bowtie) (for comparing adherence to specifications) and [Alterschema](https://github.com/sourcemeta/alterschema) (for migrating schemas to different versions) have already proven successful in addressing critical issues.

### 🚀 Next Steps :
Once we have clarity on the challenges, the next step is to make it easier for community members to create projects addressing these issues. We are planning to launch a program that provides support and benefits for such projects.

The broader discussion about building the support program is available in [Issue #521](https://github.com/json-schema-org/community/issues/521).  We encourage you to participate and share your thoughts on shaping the program to best support the Community's efforts.

###  🌱 Get Involved:
Your involvement is very important to the success of this initiative! 
Please, share your challenges, upvote the ones you resonate with, and stay tuned for the upcoming program! 🙏 

Let's build together the future of JSON Schema!

**Comment by remcohaszing** [comment]
Formally the `$schema` property as a way to reference JSON schemas.

As I understand, JSON schemas may use the `$schema` property to specify the JSON schema version used. VS Code allows to use the `$schema` property to apply any JSON schema on any JSON file. There's some ambiguity on what is valid and what isn't. A formal specification might resolve this.

See https://github.com/microsoft/vscode-json-languageservice/issues/123

**Reply by jdesrosiers** [reply]
See https://github.com/orgs/json-schema-org/discussions/473#discussioncomment-6918874 for an explanation for why this isn't likely to happen.
